plugins/woocommerce-blocks/assets/js/blocks/product-collection/block.json

```diff
@@ -34,13 +34,17 @@
 		"hideControls": {
 			"default": [],
 			"type": "array"
+		},
+		"includeInQueryContext": {
+			"type": "array"
 		}
 	},
 	"providesContext": {
 		"id": "id",
 		"queryId": "queryId",
 		"query": "query",
-		"displayLayout": "displayLayout"
+		"displayLayout": "displayLayout",
+		"includeInQueryContext": "includeInQueryContext"
```

Comment:
Nitpick: maybe `queryContextIncludes`?  Prefixing something with `include` makes me assume it's meant to be a boolean.

Reply:
Sounds good. I'll update the name to "queryContextIncludes" instead of "includeInQueryContext."